deploy-deps.sh

```diff
@@ -44,6 +46,29 @@ deploy() {
     deploy_registry
 }
 
+test_pvc_binding(){
+    PVC_RESOURCES="${script_path}/dependencies/pre-deployment-pvc-binding"
```

Comment:
please use `local` and lower case variables

Reply:
fixed